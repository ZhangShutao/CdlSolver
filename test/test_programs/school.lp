day(d1). day(d2).
schoolday(X) :- &c{schoolday(X)}, day(X).
open_school(X) :- schoolday(X), &c{open_school(X)}.
-snowday(X) :- open_school(X).
snowday(d1).