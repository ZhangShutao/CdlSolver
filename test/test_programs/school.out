default model: 1
x:day(d1),open_school(d2),day(d2),schoolday(d1),-snowday(d2),snowday(d1),schoolday(d2); y:day(d1),open_school(d2),day(d2),schoolday(d1),-snowday(d2),snowday(d1),schoolday(d2);
Satisfiable